Volume/Views/ContentView.swift

```diff
@@ -10,38 +10,110 @@ import SwiftUI
 
 struct ContentView: View {
     
-    @State private var selectedTab = 0
+    @State private var selectedTab: Tab = .home
     
     init() {
-        UITabBar.appearance().backgroundColor = UIColor(white: 250/255, alpha: 0.9)
-        UITabBar.appearance().clipsToBounds = true  // removes top border
+        let grayColor = UIColor(Color._gray)
```

Comment:
Since we use `grayColor` multiple places, can we just add it to the Color extension?

Reply:
`_gray` is already in the `Color` extension, but I'm hesitant to add the UIColor version to another extension because that introduces duplicate resources, which in turn makes it easier to accidentally have inconsistencies if we decided to tweak one but forgot about the other.